app/tests/manual/router.ts

```diff
@@ -283,24 +283,34 @@ angular
                                         group: 'Regions',
                                         set: [
                                             {
+                                                id: 13,
                                                 name: 'WEST',
                                                 value: 'WEST',
                                                 label: 'West'
                                             },
                                             {
+                                                id: 14,
                                                 name: 'EAST',
                                                 value: 'EAST',
                                                 label: 'East'
                                             },
                                             {
+                                                id: 15,
                                                 name: 'SOUTH',
                                                 value: 'SOUTH',
                                                 label: 'South'
                                             },
                                             {
+                                                id: 16,
                                                 name: 'MIDDLE',
                                                 value: 'MIDDLE',
                                                 label: 'Mid'
+                                            },
+                                            {
+                                                id: 16,
+                                                name: 'MIDDLE',
+                                                value: 'MIDDLE',
+                                                label: 'Extra Long Set Name ... Extra Long'
```

Comment:
Are these comments or is this the actual value?

Reply:
This & the other one you mentioned are just test values so if you're looking at the component in the share components test site, you can see the non-wrapping in action.